### ISSUE TITLE
Nullable<T> not supported

### ISSUE THREAD

**Issue by DanielRose** [dhelper/AssertHelper#2]
Comparisons using `Nullable<T>` cause an internal exception, ex.

    var window = new Window();
    Expect.That(() => window.ShowDialog() == true);

**Comment by dhelper** [comment]
I'll look into it - thanks for letting me know

**Comment by dhelper** [comment]
I get weird messages but not an actual exception:

I've tried your example and even simple code and it seems to work just fine (both with null and actual value).

```csharp 
bool? value = null;    

Expect.That(() => value == true); `

```

I guess I'm missing something 
- What version of AssertHelper are you using? 
- What version of C#?
- Which unit testing framework (NUnit/MSTest)?

It would greatly help if you can send me a small project showing the problem (dror.helper AT gmail).

**Comment by DanielRose** [comment]
I'm using AssertHelper 1.0.1 from NuGet, .NET 4.5.2, NUnit 2.6.4. The actual code I have is something approx. as follows:

    class Foo
    {
        public bool? Bar { get; set; }
    }

    [Test]
    public void FooTest()
    {
        var foo = new Foo();
        Expect.That(() => foo.Bar == null);
    }

**Comment by DanielRose** [comment]
Fixed with pull request #3.
